src/Licenses/Entities/License/Prediction.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace ArrowSphere\PublicApiClient\Licenses\Entities\License;
+
+use ArrowSphere\PublicApiClient\AbstractEntity;
+use ArrowSphere\PublicApiClient\Exception\EntityValidationException;
+
+/**
+ * Class ActiveSeats
+ */
+class Prediction extends AbstractEntity
+{
+    public const COLUMN_CURRENCY = 'currency';
+    public const COLUMN_UPDATE_DATE_OF_ML_SCRIPT = 'updatedAt';
+    public const COLUMN_LICENSE_REFERENCE = 'licenseRef';
+    public const COLUMN_PREDICTION_DATA = 'prediction';
```

Comment:
On specs.yaml this field is named "predictions" (with an "s")
And the constant name should reflect the value
```suggestion
    public const COLUMN_PREDICTIONS = 'predictions';
```